source/style/markdown.css

```diff
@@ -0,0 +1,7 @@
+code.refined_twitter_markdown {
+  font-family: Inconsolata, Monaco, Consolas, 'Courier New', Courier, monospace;
```

Comment:
Can you use this https://github.com/sindresorhus/modern-normalize/blob/2dc54f8e122134e09eaa3fe19b2b457c342dd426/modern-normalize.css#L116 font family instead?